Packs/MISP/Integrations/MISP_V2/MISP_V2.yml

```diff
@@ -135,6 +135,17 @@ script:
       - 'false'
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: true
+      defaultValue: 'true'
```

Comment:
what is the default behavior when this is not specified in the request?
shouldn't this be false?

Reply:
Hi, the default behaviour is 'false', meaning it will also search IOCs that are on warninglists etc. I've put it here to true, since I believe this is the more prudent behaviour, not causing false positive because of bad IOCs (common domain names, shared hosting IPs,...)

Reply:
sounds reasonable.
however, from a BC point of view - we should avoid changing the current behavior and keep the new default behavior similar to the original one.

please remove the `defaultValue` key.